dxr/static/templates/layout.html

```diff
@@ -20,54 +20,65 @@
   </head>
   <body>
 
-    <form method="get" action="{{ wwwroot }}/{{ tree }}/search" id="basic_search" class="search-box">
```

Comment:
Thanks for getting rid of that thing where there was a search-box class *and* a search-box ID. That was confusing.